h2o-docs/src/product/faq/general-troubleshooting.rst

```diff
@@ -130,14 +130,22 @@ Depending on the user-specified predictors, max_depth per tree, the number of tr
 
 **What's the best approach to help diagnose a possible memory problem on a cluster?**
 
-We've found that the best way to understand JVM memory consumption is to turn on the following JVM flags:
+We've found that the best way to understand JVM memory consumption is to turn on specific flags. These flags differ si
```

Comment:
>> We've found that the best way to understand JVM memory consumption is to turn on specific flags. These flags differ si 
Maybe change to:
We've found that the best way to understand JVM memory consumption is to turn on specific flags. These flags differ depending on your Java version.